dbms/src/Storages/DeltaMerge/Index/MinMaxIndex.cpp

```diff
@@ -510,6 +586,19 @@ RSResult MinMaxIndex::checkGreaterEqual(size_t pack_index, const Field & value,
     return RSResult::Some;
 }
 
+RSResult MinMaxIndex::checkIsNull(size_t pack_index)
+{
+    if (!(*has_value_marks)[pack_index])
+    {
+        return RSResult::All;
```

Comment:
Maybe there is no null value, but all value are deleted? I think here should return `RSResult::Some`.

Reply:
Yes, I forget the deleted case. I fix it now.

Reply:
I find another point. For our minmax index, whatever the RSResult is Some or ALL, the use_pack[i] is always true. In this case, whether we return All or return Some in this case is no difference.
But if the case is not(isnull), and then we have a pack contains deleted values and null. If we return All here, the final result of not(isnull) is None, then we don't need to read this pack. But if we return Some here, the final result of not(isnull) is Some, then we still need to read this pack.(Although the case I think may be very rare)